docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,187 @@
 # Getting started
 
-TODO
+## 1 Prerequisites
+
+- A Linux machine.
+
+## 2 Install Pebble
+
+### 2.1 Download and install the latest release
```

Comment:
In order to follow the style guide "Headings require content and should not be followed directly by a subheading", there are two solutions here:

1. Add something below this H2 heading, before the H3 heading. However, I think no extra information is needed here because it's clear this section contains two ways of doing something, and that something is already mentioned in the H2 headings. This means duplicated information.
2. Remove the H3 headings (so that no headings are stacked), but it makes the readability worse.

In this particular case, I think it's fine that nothing follows the H2 heading before the H3 heading.

There are other examples where this guide isn't strictly followed and I think it makes sense, for example, [this one](https://juju.is/docs/juju/tutorial#maintain-4). It doesn't have to say that "in this section, we will learn how to scale, perform a custom action, upgrade and observe, which are all part of maintenance".

I'm keeping it as it is, thanks.